Jenkinsfile

```diff
@@ -79,6 +79,99 @@ pipeline {
+               defaultValue: 'ubuntu20.04',
+               description: 'Image to used for Ubuntu 20 CI tests')
+        booleanParam(name: 'CI_RPM_el7_NOBUILD',
+                     defaultValue: false,
+                     description: 'Do not build RPM packages for CentOS 7')
+        booleanParam(name: 'CI_RPM_el8_NOBUILD',
+                     defaultValue: false,
+                     description: 'Do not build RPM packages for CentOS 8')
+        booleanParam(name: 'CI_RPM_leap15_NOBUILD',
+                     defaultValue: false,
+                     description: 'Do not build RPM packages for Leap 15')
+        booleanParam(name: 'CI_DEB_Ubuntu20_BUILD',
+                     defaultValue: false,
+                     description: 'Do not build DEB packages for Ubuntu 20')
+        booleanParam(name: 'CI_ALLOW_UNSTABLE_TEST',
+                     defaultValue: false,
+                     description: 'Continue testing if a previous stage is Unstable')
+        booleanParam(name: 'CI_UNIT_TEST',
+                     defaultValue: true,
+                     description: 'Run the Unit CI tests')
+        booleanParam(name: 'CI_UNIT_TEST_MEMCHECK',
+                     defaultValue: true,
+                     description: 'Run the Unit Memcheck CI tests')
+        booleanParam(name: 'CI_FUNCTIONAL_el7_TEST',
+                     defaultValue: true,
+                     description: 'Run the functional CentOS 7 CI tests')
+        booleanParam(name: 'CI_MORE_FUNCTIONAL_PR_TESTS',
+                     defaultValue: false,
+                     description: 'Enable more distros for functional CI tests')
+        booleanParam(name: 'CI_FUNCTIONAL_el8_TEST',
+                     defaultValue: true,
+                     description: 'Run the functional CentOS 8 CI tests')
+        booleanParam(name: 'CI_FUNCTIONAL_leap15_TEST',
+                     defaultValue: true,
+                     description: 'Run the functional OpenSUSE Leap 15 CI tests')
+        booleanParam(name: 'CI_FUNCTIONAL_ubuntu20_TEST',
+                     defaultValue: false,
+                     description: 'Run the functional Ubuntu 20 CI tests')
+        booleanParam(name: 'CI_RPMS_el7_TEST',
+                     defaultValue: true,
+                     description: 'Run the CentOS 7 RPM CI tests')
+        booleanParam(name: 'CI_SCAN_RPMS_el7_TEST',
+                     defaultValue: true,
+                     description: 'Run the Malware Scan for CentOS 7 RPM CI tests')
+        booleanParam(name: 'CI_small_TEST',
+                     defaultValue: true,
+                     description: 'Run the Small Cluster CI tests')
+        booleanParam(name: 'CI_medium_TEST',
+                     defaultValue: true,
+                     description: 'Run the Medium Cluster CI tests')
+        booleanParam(name: 'CI_large_TEST',
+                     defaultValue: true,
+                     description: 'Run the Large Cluster CI tests')
```

Comment:
Do we want to explicitly call these something like `CI_HW_small_TEST`, etc. so it's clear they are HW and not VM?